schemas/telemetry/crash/crash.4.schema.json

```diff
@@ -1454,6 +1454,10 @@
               "description": "<string>, Comma-separated list of enabled experimental features from about:preferences#experimental.",
               "type": "string"
             },
+            "FontName": {
+              "description": "<string>, Set before attempting to load a font to help diagnose crashes during loading.",
+              "type": "string"
+            },
```

Comment:
also needs to be added here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/d7c892dbfe46c37c44d897b8edb8b0d9f5273fec/templates/telemetry/crash/crash.4.schema.json#L88-L95

Reply:
Oh, poop. I'll add it there, too, and fix the instructions so that's clearer. Otherwise, I'll forget again. Thank you!